addons/point_of_sale/static/src/xml/Screens/ProductScreen/ProductScreen.xml

```diff
@@ -8,16 +8,10 @@
                     <OrderWidget/>
                     <div class="pads">
                         <div class="control-buttons">
-                            <t t-if="env.isMobile">
-                                <t t-if="state.displayControlButtons"
-                                   t-foreach="controlButtons" t-as="cb" t-key="cb.name">
-                                    <t t-if="!_isQuickAccessControlButton(cb.name)" t-component="cb.component" t-key="cb.name"/>
-                                </t>
```

Comment:
If there is no `firstControlButton` and no `secondControlButton` set in the config nothing than the `_displayAllControlPopup` will be show.